src/main/webapp/scripts/ou-utils.js

```diff
@@ -55,7 +55,7 @@ function addOUToJSON(ou, outputJson) {
 /*
  * Returns a list of OUs matching the name search criterion.
 */
-function searchOU(searchName) {
+function searchOU(orgUnits, searchName) {
```

Comment:
good refactor! the removal of a variable outside of the scope of a method is almost always a positive change.

Reply:
thanks! needed to do this to properly test the function too actually, haha